src/triggers.js

```diff
@@ -240,6 +241,46 @@ function logTriggerErrorBeforeHook(triggerType, className, input, auth, error) {
   });
 }
 
+export function maybeRunAfterFindTrigger(triggerType, auth, className, results, config) {
+  return new Promise(function (resolve, reject) {
+    var trigger = getTrigger(className, triggerType, config.applicationId);
+    if (!trigger) {
+      return resolve();
+    }
+    var request = getRequestObject(triggerType, auth, null, null, config);
+    var response = {
+      error: function(code, message) {
+        if (!message) {
+          message = code;
+          code = Parse.Error.SCRIPT_FAILED;
+        }
+        var scriptError = new Parse.Error(code, message);
+        return reject(scriptError);
+      }
+    };
+    logTriggerSuccessBeforeHook(triggerType, className, 'AfterFind', JSON.stringify(results), auth);
+    var resultsAsParseObjects = [];
+    for (var i=0;i<results.length;i++){
+      var result = results[i];
+      //setting the class name to transform into parse object
+      result.className=className;
+      resultsAsParseObjects.push(Parse.Object.fromJSON(result));
+    }
+    response.results = resultsAsParseObjects;
+    var triggerPromise = trigger(request, response);
+    var modifiedJSONResults = [];
+    for(var i = 0;i<response.results.length;i++){
+      modifiedJSONResults.push(response.results[i].toJSON());
+    }
+    logTriggerAfterHook(triggerType,className, JSON.stringify(modifiedJSONResults), auth);
+    if (triggerPromise && typeof triggerPromise.then === "function") {
+      return reject(new Parse.Error(Parse.Error.SCRIPT_FAILED, "Promise not supported by AfterFind"));
```

Comment:
Because the beforeFind trigger uses promises, and promises are more convenient that the callback, we should put it there too